pkg/cmd/update-tls-artifacts/generate-owners/tlsmetadatainterfaces/annotation_requirement.go

```diff
@@ -73,12 +73,12 @@ func (o annotationRequirement) generateInspectionMarkdown(pkiInfo *certs.PKIRegi
 
 	for i := range pkiInfo.CertKeyPairs {
 		curr := pkiInfo.CertKeyPairs[i]
-		if curr.InClusterLocation == nil {
+		certKeyInfo := GetCertKeyPairInfo(curr)
```

Comment:
API can be changed, yes. TLS artifact locations are being backfilled with matching info [here](https://github.com/vrutkovs/origin/blob/tls-artifacts-ondisk-todo/pkg/cmd/update-tls-artifacts/generate-owners/tlsmetadatainterfaces/helpers.go#L48-L56) to build reports easier but it could as well be the other way around